mix.exs

```diff
@@ -88,7 +88,8 @@ defmodule EMQXUmbrella.MixProject do
       {:ranch,
        github: "ninenines/ranch", ref: "a692f44567034dacf5efcaa24a24183788594eb7", override: true},
       # in conflict by grpc and eetcd
-      {:gpb, "4.19.5", override: true, runtime: false}
+      {:gpb, "4.19.5", override: true, runtime: false},
+      {:emqx_ctl, github: "emqx/emqx_ctl", tag: "0.1.0", override: true}
```

Comment:
does it have to be an external dependency ?
should it work if we add a `apps/emqx_ctl` ?

Reply:
emqx app should not need to depend on emqx_ctl?

Reply:
emqx app registers some CLI commands, so it does depend on it.